### DISCUSSION TITLE
大公司的用電量

### DISCUSSION THREAD

**Discussion by Clare0808** [1101-GS4719-C/Python-Community-GS4719-C#2030] [General]
https://technews.tw/2024/07/15/google-and-microsoft-consume-more-power-than-some-countries/

Google和微軟兩大公司的用電量還真是驚人，如果將他們的用電量分別換算成一般人的用量，兩間公司個別都可提供超過1000萬人用電！
不過還好他們都有使用再生能源，尤其是Google，很早就在實行碳中和，環保意識非常的鮮明。在發展商業的過程中，也有兼顧對地球的保護，挺令人佩服的。

**Comment by Clare0808** [comment]
後來稍微去查了一下微軟減碳以及執行負碳排的方法。減碳主要是運用新的設備冷卻技術、可再生能源，還會要求供應商公開自己的排碳量，以及推行各種雲端服務，例如計算碳足跡、遠端辦公等等；而負碳排主要是透過種樹，或者是碳捕捉技術。

**Comment by peresonacifer** [comment]
當提到這些科技公司的用電時，我馬上想到了台灣的護國神山台積電，根據之前的一份報導https://udn.com/news/story/7251/8009292
預估台積電今年用電量占全台灣8％，到2030年將會達到11～12％左右，這用電量也是相當的驚人，不過相對的他對台灣的經濟貢獻更是不可或缺。
